info.go

```diff
@@ -391,6 +397,29 @@ func (pi *ProgramInfo) Instructions() (asm.Instructions, error) {
 	return insns, nil
 }
 
+// JitedSize returns the size of the program's JIT-compiled machine code in bytes, which is the
+// actual code executed on the host's CPU. This field requires the BPF JIT compiler to be enable.
+//
+// Available from 4.13. Reading this metadata requires CAP_SYS_ADMIN or equivalent.
+//
+// The bool return value indicates whether the program has been JIT-compiled, or whether this
+// field is available.
+func (pi *ProgramInfo) JitedSize() (uint32, bool) {
+	return pi.jitedSize, pi.jitedSize > 0
+}
+
+// TranslatedSize returns the size of the program's translated instructions in bytes, after it has
+// been verified and rewritten by the kernel.
+//
+// Available from 4.13. Reading this metadata requires CAP_SYS_ADMIN or equivalent.
+func (pi *ProgramInfo) TranslatedSize() (int, error) {
+	var err error
```

Comment:
This looks a bit atypical, I would write:
```suggestion
	insns := len(pi.insns)
	if insns == 0 {
		return 0, fmt.Errorf("insufficient permissions or unsupported kernel: %w", ErrNotSupported)
	}
	return insns, nil
```